ddd-koans/Value Object/Introduction/task-info.yaml

```diff
@@ -0,0 +1,4 @@
+type: theory
+files:
+- name: src/Main.kt
+  visible: true
```

Comment:
is it possible to only show the text (task.html) instead of empty class?

Reply:
I will investigate this as if you change it to false you cannot open an introduction task of VO in Intellij.